docs/_data/prague-2020-schedule.yaml

```diff
@@ -39,9 +39,9 @@ talks_day1:
     slug: bake-a-little-documentation-love-into-your-product-matt-reiner
   - time: '18:30'
     title: Day 1 wraps up
-  - time: '19:30'
```

Comment:
This should be `docs/_data/prague-2020-schedule.yaml`

Reply:
Oh I don't know why we touched this file at all? Think we can just undo it.